src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -509,6 +513,14 @@ private Video makeVideo(Video video, RubiconVideoParams rubiconVideoParams, ExtI
                 .build();
     }
 
+    private void validateVideoSizeId(RubiconVideoParams rubiconVideoParams, String referer, String impId) {
+        final Integer videoSizeId = rubiconVideoParams != null ? rubiconVideoParams.getSizeId() : null;
+        // log only 1% of cases to monitor how often video impressions does not have size id
+        if ((videoSizeId == null || videoSizeId == 0) && System.currentTimeMillis() % 100 == 0) {
```

Comment:
Please consider using https://github.com/prebid/prebid-server-java/blob/c6c41014dd0a2627595653aba3fd7da1341d9a0e/src/main/java/org/prebid/server/log/ConditionalLogger.java#L79 or similar idea to implement logging.